Include/internal/pycore_object.h

```diff
@@ -244,6 +254,18 @@ static inline int _PyType_SUPPORTS_WEAKREFS(PyTypeObject *type) {
     return (type->tp_weaklistoffset > 0);
 }
 
+/// Method to copy objects for specified number of times inside a buffer
+static inline void
+_objects_repeat(char *dest, Py_ssize_t len_dest, Py_ssize_t len_src)
```

Comment:
The method does nothing with objects, so renaming is a good idea. Since no python elements are involved, would `_memory_repeat` be better than `_Py_memory_repeat`? Also, would it also be better to change the pointer type to `void *`, like
```
_Py_memory_repeat(void *dest, Py_ssize_t len_dest, Py_ssize_t len_src)
```
?